mycpp/gc_heap.h

```diff
@@ -188,6 +188,9 @@ class Heap {
     int n = aligned(num_bytes);
     // log("n = %d, p = %p", n, p);
 
+    // TODO(Jesse): Change to `assert(n >= sizeof(LayoutForwarded))`
+    assert(n >= 16);
```

Comment:
this the size of Obj header + the required forwarded pointer, which is the minimum size a GC object could be


comment or use constant

sizof(Obj) + size(void*)